docs/guides/installation/quick_install.md

```diff
@@ -19,4 +28,5 @@ $ bash <(curl -fsSL https://git.io/seat-installer)
  $ php /var/www/seat/artisan seat:admin:email
 ```
 
-A community provided installation video is available at the following link, visually showing the installation process: https://screencast-o-matic.com/watch/cbjUQbIcl4
+A community provided installation video is available at the following
+[link](https://screencast-o-matic.com/watch/cbjUQbIcl4), visually showing the installation process
```

Comment:
That video is done by me with SeAT 2.0. I would delete it and replace it by qu1ckks version